plenum/server/monitor.py

```diff
@@ -143,6 +188,10 @@ def __init__(self, name: str, Delta: float, Lambda: float, Omega: float,
         # protocol instance
         self.numOrderedRequests = []  # type: List[Tuple[int, int]]
 
+        # List of throughputs for replicas. Index is a instId and value is a instance of
```

Comment:
Is it really needed in RequestTimeTracker? Probably placing it in monitor would better separate responsibilities?>